add models commit

hailaSE opened the pull request.

hailaSE committed:
add models commit

Ahmad-Mohammad-Kouja commented on app/Models/Item.php:
```diff
@@ -0,0 +1,61 @@
+        return $this->hasMany(Offer::class);
+    }
+
+    public function item_orders(): HasMany
```
its itemOrders not item_orders

Ahmad-Mohammad-Kouja commented on app/Models/ItemOrder.php:
```diff
@@ -0,0 +1,33 @@
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class ItemOrder extends Model
```
can we rename this to orderDetail instead of itemOrder

Ahmad-Mohammad-Kouja commented on app/Models/Notification.php:
```diff
@@ -0,0 +1,28 @@
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\MorphTo;
+
+class Notification extends Model
```
can we remove this its already Laravel has built in notification class

Ahmad-Mohammad-Kouja commented on app/Models/Order.php:
```diff
@@ -0,0 +1,54 @@
+        'time_delivery' => 'datetime'
+    ];
+
+    public function item_orders(): HasMany
```
itemOrders

Ahmad-Mohammad-Kouja commented on app/Models/Store.php:
```diff
@@ -0,0 +1,60 @@
+    protected $table = 'stores';
+
+    protected $fillable = [
+        'admin_id',
```
can we remove the admin id here
same for the admin realation

Ahmad-Mohammad-Kouja commented on app/Models/Store.php:
```diff
@@ -0,0 +1,60 @@
+        return $this->belongsTo(Admin::class);
+    }
+
+    public function contact_infos(): HasMany
```
contactInfos

Ahmad-Mohammad-Kouja commented on app/Models/Store.php:
```diff
@@ -0,0 +1,60 @@
+        return $this->hasMany(Feedback::class);
+    }
+
+    public function item_stores(): HasMany
```
itemStores

Ahmad-Mohammad-Kouja commented on app/Models/User.php:
```diff
@@ -42,4 +49,45 @@ class User extends Authenticatable
+        return $this->hasMany(Wishlist::class);
+    }
+
+    public function user_products(): HasMany
```
this should be products

Ahmad-Mohammad-Kouja commented on app/Models/Wishlist.php:
```diff
@@ -0,0 +1,32 @@
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class Wishlist extends Model
```
here also should be a store id

Ahmad-Mohammad-Kouja commented on app/Models/Store.php:
```diff
@@ -0,0 +1,60 @@
+        'name',
+        'city',
+        'location',
+        'delivery_fee',
```
can we remove the delivery fee and location

hailaSE committed:
fix models

hailaSE committed:
fix orderDetail model

fix name attribute  to order_details

hailaSE committed:
update fillable attribute in work model

add store_id

hailaSE committed:
fix payment relation in order model

change the relation to hasOne

hailaSE committed:
fix order relation in address model

change the relation to hasOne

hailaSE committed:
add store relation in City model

hailaSE committed:
add city relation in Store model

hailaSE committed:
fix wishlist model  and itemstore model

add relation between wishlist and itemstore
remove relation between store and wishlist
remove relation between item and wishlist

hailaSE committed:
fix model user
